cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformResources.java

```diff
@@ -170,10 +170,14 @@ public CloudNetworks networks(ExtendedCloudCredential cloudCredential, Region re
         String networkId = null;
         List<String> subnetIds = new ArrayList<>();
         String sharedProjectId = null;
+        String customAvailabilityZone;
         if (filters != null) {
             networkId = filters.getOrDefault("networkId", null);
             subnetIds = getSubnetIds(filters);
             sharedProjectId = filters.getOrDefault("sharedProjectId", null);
+            customAvailabilityZone = filters.getOrDefault(GcpStackUtil.CUSTOM_AVAILABILITY_ZONE, "");
+        } else {
+            customAvailabilityZone = "";
```

Comment:
yes, I thought the same, but somehow it gave me a compile error, because the variable is used in lambda and it should be effectively final..
![image](https://github.com/hortonworks/cloudbreak/assets/2011396/9955f2d5-bdc0-4391-9f63-f5ac34d6b6d5)